.github/workflows/wheels.yml

```diff
@@ -32,16 +40,43 @@ jobs:
         os:
           - ubuntu-latest
           - macos-latest
-          # - windows-latest
+          - windows-latest
 
     steps:
     - uses: actions/checkout@v2
       with:
         submodules: recursive
 
+    - name: Cache Conan
+      id: cache-conan
+      uses: actions/cache@v2
+      if: matrix.os == 'windows-latest'
+      with:
+        path: |
+          conan_build
+          conan_data
+        key: conan-${{ matrix.os }}-${{ hashFiles('conanfile.txt') }}
+
+    - name: Set up Windows Python ${{ matrix.python_version }}
+      uses: actions/setup-python@v2
+      if: matrix.os == 'windows-latest' && steps.cache-conan.outputs.cache-hit != 'true'
+      with:
+        python-version: '3.x'
+
+    - name: Install conan for Windows
+      if: matrix.os == 'windows-latest' && steps.cache-conan.outputs.cache-hit != 'true'
+      run: |
+        pip install pip --upgrade
+        pip install conan
+        conan profile new default --detect
+        conan profile update "settings.compiler=Visual Studio" default
+        conan profile update "settings.compiler.version=16" default
+        conan config set "storage.path=$env:GITHUB_WORKSPACE/conan_data"
+        conan install --build=openssl --install-folder conan_build .
```

Comment:
I took this back instead of the conan_install.bat. we need the `$env:GITHUB_WORKSPACE`. after this worked in your repo with all jobs active I'll try again with an env var in the .bat file. it _should_ work, but honestly if it doesn't on the first try, I'll leave this as it is now..

Reply:
ran through, but I think we can leave as it is in any case and accept that windows is just bääh;)